client/lib/preferences-helper/preference.js

```diff
@@ -39,19 +39,20 @@ class Preference extends Component {
 		}
 
 		return (
-			<div>
-				<div className="preferences-helper__preference-header">
+			<tr>
+				<td className="preference__unset">
 					<button
-						className="preferences-helper__unset"
 						onClick={ partial( unsetPreference, name, null ) }
 						title={ translate( 'Unset Preference' ) }
 					>
-						{ 'X' }
+						<span role="img" aria-label={ translate( 'Unset Preference' ) }>
```

Comment:
I actually think we should make all those strings literal, and non-localized. This being a dev tool, I'm not sure it makes sense to be localized.